WeScan.xcodeproj/project.pbxproj

```diff
@@ -865,7 +865,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "./Submodules/WeTransfer-iOS-CI/SwiftLint/swiftlint.sh\n";
+			shellScript = "if [ -z \"$CI\" ]; then\n    if [ \"${CONFIGURATION}\" == \"Debug\" ]; then\n        if test -d \"/opt/homebrew/bin/\"; then\n            PATH=\"/opt/homebrew/bin/:${PATH}\"\n        fi\n\n        export PATH\n        swiftlint --config \"${SRCROOT}/../Submodules/WeTransfer-iOS-CI/BuildTools/.swiftlint.yml\"\n    else\n        echo \"Info: As we're not building for Debug, no SwiftLint is running.\"\n    fi\nfi\n\n";
```

Comment:
@AvdLee I couldn't get swiftlint to work. Can you have a look at that?

Reply:
This should be the trick; copied from UINotifications:
https://github.com/WeTransfer/UINotifications/pull/67/files#diff-e1ff36b93cdf80522da2ae2618a67ba627ce6fa7a6f0d8d9a93019468234479bR179